src/main/java/DataBase/DatasourceConfig.java

```diff
@@ -20,8 +20,8 @@ public class DatasourceConfig {
     /*
      * Change database url and login information depending on your database
      * */
-    private static String dbUrl = "jdbc:postgresql://localhost/dbname";
-    private static String user = "username";
+    private static String dbUrl = "jdbc:postgresql://localhost/postgres";
+    private static String user = "postgres";
     private static String password = "password";
```

Comment:
Kannst Du bitte nochmal prüfen, ob in der Anleitung steht, dass "password" als Passwort für die Datenbank gesetzt werden soll? Wenn nicht, müsste man dazu noch einen Satz ergänzen